ParkingTicketFrontEnd/Extensions/ParkingOffenseFactory.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+using ParkingTicketLogic;
+
+namespace ParkingTicketFrontEnd.Extensions
+{
+    public class ParkingOffenseFactory
```

Comment:
Let's name this to be a bit better. How about something like ParkingList SelectListGenerator

Reply:
Not a bad idea I will make that change on Wednesday

Reply:
This is fixed and now it is now as EmumerationExtensionMethods.SelectListFor()